aries_vcx/src/error.rs

```diff
@@ -0,0 +1,550 @@
+use std::cell::RefCell;
```

Comment:
I think this file can be reuse from `libvcx`? (it's currently duplicated)

Reply:
Not sure that's a good pattern. I think each library should have its own error types specific to its use-cases, purposes, concerns, APIs, and flows. Else, adding a new error type would require modification of a different library.